llvm/test/CodeGen/AMDGPU/ctlz_zero_undef.ll

```diff
@@ -2621,13 +2579,11 @@ define <2 x i7> @v_ctlz_zero_undef_v2i7(<2 x i7> %val) {
 ; GFX9-GISEL-LABEL: v_ctlz_zero_undef_v2i7:
 ; GFX9-GISEL:       ; %bb.0:
 ; GFX9-GISEL-NEXT:    s_waitcnt vmcnt(0) expcnt(0) lgkmcnt(0)
-; GFX9-GISEL-NEXT:    v_and_b32_e32 v0, 0x7f, v0
-; GFX9-GISEL-NEXT:    v_and_b32_e32 v1, 0x7f, v1
-; GFX9-GISEL-NEXT:    v_ffbh_u32_e32 v0, v0
-; GFX9-GISEL-NEXT:    v_ffbh_u32_e32 v1, v1
-; GFX9-GISEL-NEXT:    v_subrev_u32_e32 v0, 25, v0
-; GFX9-GISEL-NEXT:    v_subrev_u32_e32 v1, 25, v1
-; GFX9-GISEL-NEXT:    s_setpc_b64 s[30:31]
+; GFX9-GISEL-NEXT:    v_lshrrev_b32_e32 v0, 25, v0
+; GFX9-GISEL-NEXT:  	v_lshrrev_b32_e32 v1, 25, v1
+; GFX9-GISEL-NEXT:  	v_ffbh_u32_e32 v0, v0
+; GFX9-GISEL-NEXT:  	v_ffbh_u32_e32 v1, v1
+; GFX9-GISEL-NEXT:  	s_setpc_b64 s[30:31]
```

Comment:
Something weird happened with the indentation in some of these functions, no idea how that's possible